### DISCUSSION TITLE
Flash of error before page load

### DISCUSSION THREAD

**Discussion by RickCogley** [retypeapp/retype#212] [Q&A] [answered]
Using github pages to deploy, when I load my retype site (https://help.esolia.pro) pages before they get cached by the browser (seems like the cache expired overnight), I see this error flash for a split second, in the upper left of the browser screen, before the page content loads:

```
FATAL: corrupted website configuration
```

The site is working and I get no fatal errors locally with `retype watch --verbose`, but the error is disconcerting, so I wanted to ask if anyone knows about this...

**Comment by AndreyChechel** [comment, marked as answer]
@RickCogley thanks for letting us know about that issue! 
We'll investigate right away.

Looks like this problem could be introduced in `v1.11` - Retype now does more work to ensure the correctness/integrity of the configuration and scripts it loads. When an updated website is deployed (though the github action, or manually), browsers are instructed to re-fetch scripts and the website data.

I guess, for some reason there was a moment of inconsistency, when some scripts have been already updated, but some have not - Retype detected and reported that. It's likely that the issue exists only for the websites that have been just updated to `v1.11`, i.e. subsequent updates will not generate that error.

Do you use any custom setting for [cache.busting](https://retype.com/configuration/project/#busting) configuration?

**Reply by RickCogley** [reply]
Didn't specify anything wrt cache busting.

**Reply by AndreyChechel** [reply]
I've created the issue #226 to track this defect.
The problem is reproducible even when both old and new website versions are built with `Retype v1.11`.

**Comment by geoffreymcgill** [comment]
This issue has been fixed and included in the Retype [`v1.11.1`](https://github.com/retypeapp/retype/releases/tag/v1.11.1) release.

Hope this helps.

**Reply by RickCogley** [reply]
thanks!
